apps/aecore/src/txs/aec_tx.erl

```diff
@@ -78,3 +106,26 @@ check_single(Tx, Trees, Height) ->
 process_single(Tx, Trees, Height) ->
     Mod = tx_dispatcher:handler(Tx),
     Mod:process(Tx, Trees, Height).
+
+-spec grant_fee_to_miner(list(tx()), trees(), height(), non_neg_integer()) ->
+                                trees().
+grant_fee_to_miner([], Trees, _Height, 0) ->
+    lager:debug("No transactions in genesis block"),
+    Trees;
+grant_fee_to_miner(Txs, Trees0, Height, TotalFee) ->
+    %% Consider creation of aec_accounts_service,
+    %% which will take state trees, height and list of (pubkey, operation) pairs
+    %% (valuable also during txs processing)
+    MinerPubkey = coinbase_tx_account_pubkey(Txs),
+    AccountsTrees0 = aec_trees:accounts(Trees0),
+
+    {ok, Account0} = aec_accounts:get(MinerPubkey, AccountsTrees0),
+    {ok, Account} = aec_accounts:earn(Account0, TotalFee, Height),
+
+    {ok, AccountsTrees} = aec_accounts:put(Account, AccountsTrees0),
+    Trees = aec_trees:set_accounts(Trees0, AccountsTrees),
+    Trees.
```

Comment:
@uwiger @gorillainduction mind that there is no transaction representing update to miners account by sum of all fees

Reply:
I see that coinbase tx has the responsibility of identifying the account of the miner, and contains no amont.
@zp-sd Could you please add comment in [`aec_coinbase_tx:process`](https://github.com/aeternity/epoch/pull/322/files#diff-933a22fcb74313128547fe0436956dd9R37) mentioning that the earning of the total of the non-coinbase tx fees is not computed inside the coinbase transaction but rather at the end of the application of all transactions in the block?

Reply:
@lucafavatella Coinbase tx still adds `block_mine_reward` to miner's account, which is driven by "governance".